cmd/analyse.go

```diff
@@ -60,5 +60,9 @@ func runAnalyse(cmd *cobra.Command, args []string) {
 		Host:            viper.GetString("host"),
 		RawManifestFile: manifestFile,
 	}
-	sa.StackAnalyses(requestParams)
+	analysesResult := sa.StackAnalyses(requestParams)
```

Comment:
I feel it is better to have a flag for the concise summary.

e.g. --summary

Reply:
No, Summary is default behaviour as per ask. No flags
`--verbose` aka Detailed report is with flag.